paket.dependencies

```diff
@@ -5,6 +5,7 @@ nuget Argu 3.7
 nuget FSharp.Compiler.Service 11.0.9 framework: >= net45
 nuget FSharp.Compiler.Service.ProjectCracker 11.0.9
 nuget Dotnet.ProjInfo 0.7.4
+nuget Sln 0.2.0
```

Comment:
is not 1.0 yet, so i can publish new version and i dont guarantee the api compatibility between minor.
just to avoid issues for others, while updating deps.